app/src/main/java/com/helio/app/networking/HubService.java

```diff
@@ -82,4 +82,7 @@
 
     @DELETE("/light/unregister/{light_sensor_id}")
     Call<ResponseBody> deleteLightSensor(@Path("light_sensor_id") int id);
+
+    @GET("/motor/get_all")
+    Call<List<Motor>> getNetworkStatus();
```

Comment:
We could ask @AlesRessler if we could add a dedicated connectivity check endpoint? Something like `/ping/`. This works well though